xml/klp.xml

```diff
@@ -24,382 +23,315 @@
-   they happen while the kernel is running in any case.
-  </para>
+  <informalfigure>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="klp.png" width="80%" format="PNG"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="klp.png" width="100%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </informalfigure>
 
   <para>
-   Callers are never patched. Instead, the callee's NOPs are replaced by a JMP
-   to the new function. JMP instructions remain forever. This takes care of
-   function pointers, including in structures, and does not require saving any
-   old data for the possibility of un-patching.
+   It is possible to have multiple versions of the kernel package installed
+   along with their live patches. These packages do not conflict. You can
+   install updated kernel packages along with live patches for the running
+   kernel. In this case, you may be prompted to reboot the system. Users with
+   &slea; Live Patching subscription are eligible for technical support as long
+   as there are live patch updates for the running kernel (see
+   <xref linkend="sec-klp-lifecycle" />).
   </para>
 
   <para>
-   However, these steps alone would not be good enough: since the functions
-   would be replaced non-atomically, a new fixed function in one part of the
-   kernel could still be calling an old function elsewhere or vice versa. If
-   the semantics of the function interfaces changed in the patch, chaos would
-   ensue.
+   With &klpa; activated, every kernel update comes with a live patch package.
+   This live patch does not contain any fixes and serves as a seed for future
+   live patches for the corresponding kernel. These empty seed patches are
+   called <literal>initial patches</literal>.
   </para>
 
-  <para>
-   Thus, until all functions are replaced, &klp; uses an approach based on
-   trampolines and similar to RCU (read-copy-update), to ensure a consistent
-   view of the world to each user space thread, kernel thread and kernel
-   interrupt. A per-thread flag is set on each kernel entry and exit. This way,
-   an old function would always call another old function and a new function
-   always a new one. Once all processes have the "new universe" flag set,
-   patching is complete, trampolines can be removed and the code can operate at
-   full speed without performance impact other than an extra-long jump for each
-   patched function.
-  </para>
- </sect1>
-
-  <sect1 xml:id="sec-klp-install-patch-activate">
-   <title>Activation of SLE live patching</title>
+  <sect2 xml:id="sec-klp-scope">
+   <title>&klp; scope</title>
    <para>
-    To activate &slea; Live Patching on your system, follow these steps:
+    The scope of &slea; Live Patching includes fixes for SUSE Common
+    Vulnerability Scoring System (CVSS; SUSE CVSS is based on the CVSS v3.0
+    system) level 7+ vulnerabilities and bug fixes related to system stability
```

Comment:
This is a very long parenthetical note. How about moving it into its own sentence, after this one?